ihmc-perception/src/main/java/us/ihmc/perception/filters/CUDAFlyingPointsFilter.java

```diff
@@ -0,0 +1,60 @@
+package us.ihmc.perception.filters;
+
+import org.bytedeco.cuda.cudart.CUstream_st;
+import org.bytedeco.cuda.cudart.dim3;
+import org.bytedeco.cuda.global.cudart;
+import org.bytedeco.opencv.global.opencv_core;
+import org.bytedeco.opencv.opencv_core.GpuMat;
+import us.ihmc.perception.cuda.CUDAKernel;
+import us.ihmc.perception.cuda.CUDAProgram;
+
+import java.net.URL;
+
+import us.ihmc.perception.cuda.CUDAStreamManager;
+
+
+public class CUDAFlyingPointsFilter
+{
+   private static final int BLOCK_SIZE_XY = 16;
+   
+   private final CUDAKernel flyingPointFilterKernel;
+   private final CUDAProgram flyingPointFilterCUDAProgram;
+   private final CUstream_st stream;
+
+   public CUDAFlyingPointsFilter()
+   {
+      stream = CUDAStreamManager.getStream();
```

Comment:
You might want to pass stream into the applyFilter method instead of storing it as a field